package.json

```diff
@@ -42,18 +42,25 @@
 			"dist/ParallelApi.credentials.js"
 		],
 		"nodes": [
-			"dist/Parallel.node.js"
+			"dist/Parallel.node.js",
+			"dist/ParallelTrigger.node.js"
 		]
 	},
 	"devDependencies": {
+		"@types/node": "^24.5.2",
 		"@typescript-eslint/parser": "~8.32.0",
 		"eslint": "^8.57.0",
 		"eslint-plugin-n8n-nodes-base": "^1.16.3",
 		"gulp": "^5.0.0",
+		"n8n-workflow": "^1.82.0",
 		"prettier": "^3.5.3",
 		"typescript": "^5.8.2"
 	},
 	"peerDependencies": {
 		"n8n-workflow": "*"
+	},
+	"dependencies": {
+		"axios": "^1.12.2",
```

Comment:
runtime dependencies aren't allowed, and I see they aren't used. you can probably just remove these.